templates/strapi/files/README.md

```diff
@@ -0,0 +1,63 @@
+# Strapi template for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/strapi/.platform.template.yaml&utm_content=strapi&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a Strapi backend for Platform.sh.
```

Comment:
Worth noting here that it does not include a front-end, so it's going to not look like much after install.

Reply:
(And in the template.yaml file.)